testdata/issue74.h

```diff
@@ -0,0 +1,42 @@
+struct perf_event_attr {
```

Comment:
This code has copyright, but it's not reproduced here. Even then, we would need to include the respective LICENSE. Perfectly possible, of course.

Or you can just write your own `issue74.h` from scratch, if you like.